maven-java/kalix-maven-plugin/src/main/scala/kalix/RunMojo.scala

```diff
@@ -49,18 +48,7 @@ object RunMojo {
         element(name("argument"), "-classpath"),
         element(name("classpath")))
 
-    val loggingArgs: Seq[Element] = {
-      val logConfigFile = new File(logConfig.trim)
-      if (logConfigFile.exists) {
-        log.info(s"Using logging configuration file: '$logConfigFile' ")
-        // when using SpringBoot, logback config is passed using logging.config
-        element(name("argument"), "-Dlogging.config=" + logConfig) ::
-        element(name("argument"), "-Dlogback.configurationFile=" + logConfig) :: Nil
-      } else {
-        log.warn(s"Dev mode logging configuration file '$logConfig' not found")
-        List.empty
```

Comment:
we were previously doing this kind of hack in the plugin. We don't need it anymore as the JoranConfigurator will do it as soon as LoggerFactory is used. And this is valid for both java and scala.